src/viam/examples/dial/example_dial.cpp

```diff
@@ -1,28 +1,29 @@
-#include <common/v1/common.pb.h>
-#include <grpcpp/channel.h>
-#include <grpcpp/client_context.h>
-#include <grpcpp/grpcpp.h>
-#include <grpcpp/support/status.h>
-#include <robot/v1/robot.grpc.pb.h>
-#include <robot/v1/robot.pb.h>
-#include <unistd.h>
-
-#include <boost/optional.hpp>
 #include <cstddef>
 #include <functional>
 #include <iostream>
 #include <memory>
 #include <ostream>
 #include <string>
+#include <unistd.h>
 #include <vector>
 
-#include <components/generic/client.hpp>
-#include <robot/client.hpp>
-#include <robot/service.hpp>
-#include <rpc/dial.hpp>
+#include <boost/optional.hpp>
+#include <grpcpp/channel.h>
+#include <grpcpp/client_context.h>
+#include <grpcpp/grpcpp.h>
+#include <grpcpp/support/status.h>
+
+#include <viam/api/common/v1/common.pb.h>
+#include <viam/api/robot/v1/robot.grpc.pb.h>
+#include <viam/api/robot/v1/robot.pb.h>
+
+#include <viam/sdk/components/generic/client.hpp>
```

Comment:
This is the shape of includes for all SDK headers, now and in all our other sources.